website_sale_stock_provisioning_date/models/product_product.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    def _get_next_provisioning_date(self):
+        domain = [
+            ("company_id", "=", self.env.user.company_id.id),
```

Comment:
you can use self.env.company directly in v13.
I have a doubt. Should the company be user's company or company of website??
What do you think?

Reply:
I think it should be the website company to show the closest date on which the company corresponding to the website will receive the product. I'm going to change that condition of the domain then. You agree with me?

Reply:
perfect